pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -58,24 +65,32 @@
 , tabulate
 , tenacity
 , termcolor
+, typing-extensions
 , unicodecsv
 , werkzeug
 , pytestCheckHook
 , freezegun
 , mkYarnPackage
 }:
 let
-  version = "2.3.3";
+  version = "2.3.4";
 
   airflow-src = fetchFromGitHub rec {
     owner = "apache";
     repo = "airflow";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-N+6ljfSo6+UvSAnvDav6G0S49JZ1VJwxmaiKPV3/DjA=";
+    # Required because the GitHub archive tarballs don't appear to include tests
```

Comment:
?!

Can confirm this, and am kinda shocked. It looks like it's possible to "overwrite" the zip file that will be served for a particular tag by having a release asset named similarly.

Reply:
Thanks for confirming, thought I was going mad!